modin/engines/ray/wrapper.py

```diff
@@ -0,0 +1,15 @@
+import ray
+
+
+class RayWrapper:
+    @classmethod
+    def deploy(cls, func, num_return_vals, kwargs):
+        return deploy_ray_func.remote(args=(kwargs), num_return_vals=num_return_vals)
+
+
+@ray.remote
+def deploy_ray_func(self, func, args):  # pragma: no cover
+    for k, v in args.items():
```

Comment:
This loop isn't necessary in our case.

Reply:
Some of the values in the dictionary are object ids rather than objects - will ray automatically look through the dict values and do a ray.get?

Reply:
No Ray will not do this, but nothing should be an object ID in the dictionary

Reply:
In the original code, since its ray specific, it does a put which is why this happens. I'll remove the put so that it doesn't need this?